src/devices/Bmx280/samples/Bme280.sample.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <EnableDefaultItems>false</EnableDefaultItems>
```

Comment:
NIT: we don't need this property on sample projects as we don't mind in this case if bindings include *.cs to the Compile item.

Reply:
I have 2 projects in the same directory, so I guess it's needed. And ```async main``` is used so need one of the latest version of language.

Reply:
for latest compiler  you can go ahead and use that everywhere